tools/samba/illumina_suite/figaro.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="figaro" name="Figaro" version="@TOOL_VERSION_FIGARO@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>Identify optimal optimizing microbiome rRNA gene trimming parameters for DADA2</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="figaro_xref"/>
+    <expand macro="figaro_edam"/>
+    <requirements>
+        <container type="docker">zymoresearch/figaro</container>
+    </requirements>
+    <required_files>
+        <include type="literal" path="macro.xml"/>
+    </required_files>
+    <command detect_errors="exit_code"><![CDATA[
+        cd /opt/figaro/figaro
+        ## Run FIGARO
+        python3 figaro.py -i '$raw_data':/data/input -o '$figaro_output':/data/output -a '$amplicon_length' -f '$f_primer' -r '$r_primer'
```

Comment:
So this is launching the docker image with all the right params. Now we need to think of way to have the data of `$raw_data` stored within a directory or a workaround because it can't read them otherwise